### PR TITLE
M16A4 has full auto

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Ren bugged me for it
it has full auto

## Why It's Good For The Game
uhh QOL I guess

## Changelog
:cl:
balance: m16 has full auto 
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by spookydonut** [comment]
@Pariah919 stop closing/reopening prs, it won't fix travis errors.

If there are problems with travis jobs I will manually solve them without you having to do anything.

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -397,7 +397,7 @@
 						/obj/item/attachable/scope/mini)
 
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_LOAD_INTO_CHAMBER|GUN_AMMO_COUNTER
-	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_BURSTFIRE, GUN_FIREMODE_AUTOBURST)
+    gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_BURSTFIRE, GUN_FIREMODE_AUTOBURST, GUN_FIREMODE_AUTOMATIC)
```
Comment (spookydonut) [inline code/modules/projectiles/guns/rifles.dm:400]:
```suggestion
	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_BURSTFIRE, GUN_FIREMODE_AUTOBURST, GUN_FIREMODE_AUTOMATIC)
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-4). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**